### PR TITLE
[DC-2022] Add latest sponsors and new sponsor: Zoho

### PR DESCRIPTION
**Pull request by pburkholder**
.
[DC-2022] Add latest sponsors and new sponsor: Zoho

### CODE REVIEW

**Comment by pburkholder** [comment]
Please hold. Tremolo doesn't have a data file yet...

**Comment by pburkholder** [comment]
Looks good. Please ship!

Preview at https://deploy-preview-11485--devopsdays-web.netlify.app/events/2022-washington-dc/welcome/

File: data/sponsors/tremolo.yml [hunk 1]
```diff
@@ -0,0 +1,3 @@
+name: "zoho"
+url: "https://zoho.com"
+twitter: "zoho"
```
Comment (adrianmoisey) [inline data/sponsors/tremolo.yml:3]: These are the details for zoho, I think they need to be for tremolo.

**Comment by pburkholder** [comment]
Thanks for the catch. Fixed now!

**Pull request merged**